plugins/destination/neo4j/client/write.go

```diff
@@ -20,7 +20,7 @@ func (c *Client) WriteTableBatch(ctx context.Context, tableName string, msgs mes
 		return err
 	}
 
-	session := c.LoggedSession(ctx, neo4j.SessionConfig{})
+	session := c.Session(ctx, neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite})
```

Comment:
What is the default access mode? does this require users to potentially change credentials or permissions?

Reply:
Default is `0` = `write`:
https://pkg.go.dev/github.com/neo4j/neo4j-go-driver/v5@v5.13.0/neo4j#AccessMode

Reply:
The change, however, requires users to have `CREATE INDEX` & `DROP INDEX` privileges